src/widgets/dashboardOverview.js

```diff
@@ -28,13 +28,40 @@ export default async function dashboardOverview(scopes) {
   const inPerson = duration.filter((report) => report.deliveryMethod.toLowerCase() === 'in-person').length
     .toString();
 
+  // Get Region.
+  const region = query['region.in'];
+
+  // Get Start and End Date.
+  const dateRange = query['startDate.win'];
+  if (!dateRange) {
+    return null;
+  }
+  const dates = dateRange.split('-');
+
+  let startDte;
+  let endDte;
+
+  // Check if we have a Start Date.
+  if (dates.length > 0) {
+    // eslint-disable-next-line prefer-destructuring
+    startDte = dates[0];
+  }
+
+  // Check if we have and End Date.
+  if (dates.length > 1) {
+    // eslint-disable-next-line prefer-destructuring
+    endDte = dates[1];
+  }
+
+  const participantsWhere = `WHERE "regionId" in (${region !== undefined ? region : '0'})  AND "status" = '${REPORT_STATUSES.APPROVED}' AND "startDate" >= '${startDte}' AND "startDate" <= '${endDte}'`;
```

Comment:
Looking for activity reports with `region_id = 0` will give us zero activity reports.

Reply:
Right we shouldn't have a case where region is undefined I would think. Do you think this needs to be changed to a different value?

Reply:
I think I would expect a request with an undefined region to throw an error. And if we throw an error, I would prefer a fast fail around line 32.

Reply:
Similar to my comment https://github.com/adhocteam/Head-Start-TTADP/pull/376/files#r682964384, I don't think it's a good idea to check params so late in the game. We should check params up top and do a fast fail whenever possible. Again, I don't want to hold up this PR. This can be future tech debt.